woocommerce/woocommerce
# Rename woocommerce-legacy-assets

### All Submissions:






### Changes proposed in this Pull Request:

Aligning with #33460, this pull request renames the `woocommerce-legacy-assets` package to `woocommerce/client/legacy`. This also moved the `plugins/woocommerce/legacy` folder into `plugins/woocommerce/client/legacy` for consistency with the name.
### How to test the changes in this Pull Request:

1. Run `pnpm -- turbo run build --filter=woocommerce/client/legacy`
2. Make sure WooCommerce still works

### Other information:




### FOR PR REVIEWER ONLY:
